backend/satellite_tools/mgr-update-pkg-extra-tags.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/python3
```

Comment:
When this is backported to 3.2 as well, you need to remember it and change this to python2